pkg/cmd/reset.go

```diff
@@ -63,6 +64,7 @@ It's possible to source information about hosts from Terraform output, using the
 	}
 
 	cmd.Flags().BoolVarP(&ropts.DestroyWorkers, "destroy-workers", "", true, "destroy all worker machines before resetting the cluster")
+	cmd.Flags().BoolVarP(&ropts.RemovePackages, "remove-packages", "", false, "remove kubernetes packages after resetting the cluster")
```

Comment:
I'm concerned about the flag name. Packages may be a bad word here because we don't use package management solution on all platforms. For example, on CoreOS we download and install binaries manually. Technically, `remove-binaries` could work better. Any opinions?

Reply:
`remove-binaries` sgtm

Reply:
Done. :100: